.devcontainer/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM codercom/enterprise-base:ubuntu
+
+ENV GOROOT /usr/local/go
+ENV GOPATH /home/coder/go
+ENV PATH $GOPATH/bin:$GOROOT/bin:$PATH
+
+ENV DEBIAN_FRONTEND noninteractive
+
+USER root
+
+# Install apt packages
+RUN add-apt-repository ppa:deadsnakes/ppa &&\
+    apt-get update &&\
+    apt-get install -y nano python3.9 &&\
+    # Cleanup
+    apt-get clean &&\
+    rm -rf /var/lib/apt/lists/*
+
+# Golang Install 1.19.4
```

Comment:
I don't know if this is a big deal but Armada is actually still on 1.18.

Reply:
I think I install both 1.19 and 1.18 here?

I could maybe install a program to make it easier to switch between them... That would be a nice addition

Reply:
If it works then I’m happy. Just noticed it so feel free to ignore.